ViewLayer/CompactUI/CompactUI.h

```diff
@@ -0,0 +1,193 @@
+    void setMppt7VoltageIn(double);
+    void setMppt7VoltageOut(double);
+
+    double getMppt1CurrentIn();
+    double getMppt1CurrentOut();
+    double getMppt2CurrentIn();
+    double getMppt2CurrentOut();
+    double getMppt3CurrentIn();
+    double getMppt3CurrentOut();
+    double getMppt4CurrentIn();
+    double getMppt4CurrentOut();
+    double getMppt5CurrentIn();
+    double getMppt5CurrentOut();
+    double getMppt6CurrentIn();
+    double getMppt6CurrentOut();
+    double getMppt7CurrentIn();
+    double getMppt7CurrentOut();
+
+    double getMppt1VoltageIn();
+    double getMppt1VoltageOut();
+    double getMppt2VoltageIn();
+    double getMppt2VoltageOut();
+    double getMppt3VoltageIn();
+    double getMppt3VoltageOut();
+    double getMppt4VoltageIn();
+    double getMppt4VoltageOut();
+    double getMppt5VoltageIn();
+    double getMppt5VoltageOut();
+    double getMppt6VoltageIn();
+    double getMppt6VoltageOut();
+    double getMppt7VoltageIn();
+    double getMppt7VoltageOut();
+
+    QLabel& setSetSpeed();
+    QLabel& setSetCurrent();
+    QLabel& setActualSpeed();
+    QLabel& setBusCurrent();
+    QLabel& setBusVoltage();
+    QLabel& setBatteryCurrent();
+    QLabel& setBatteryVoltage();
+
+    QLabel& setBatteryCMU1Temp();
+    QLabel& setBatteryCMU1Cell1Voltage();
+    QLabel& setBatteryCMU1Cell2Voltage();
+    QLabel& setBatteryCMU1Cell3Voltage();
+    QLabel& setBatteryCMU1Cell4Voltage();
+    QLabel& setBatteryCMU1Cell5Voltage();
+    QLabel& setBatteryCMU1Cell6Voltage();
+    QLabel& setBatteryCMU1Cell7Voltage();
+    QLabel& setBatteryCMU1Cell8Voltage();
+
+    QLabel& setBatteryCMU2Temp();
+    QLabel& setBatteryCMU2Cell1Voltage();
+    QLabel& setBatteryCMU2Cell2Voltage();
+    QLabel& setBatteryCMU2Cell3Voltage();
+    QLabel& setBatteryCMU2Cell4Voltage();
+    QLabel& setBatteryCMU2Cell5Voltage();
+    QLabel& setBatteryCMU2Cell6Voltage();
+    QLabel& setBatteryCMU2Cell7Voltage();
+    QLabel& setBatteryCMU2Cell8Voltage();
+
+    QLabel& setBatteryCMU3Temp();
+    QLabel& setBatteryCMU3Cell1Voltage();
+    QLabel& setBatteryCMU3Cell2Voltage();
+    QLabel& setBatteryCMU3Cell3Voltage();
+    QLabel& setBatteryCMU3Cell4Voltage();
+    QLabel& setBatteryCMU3Cell5Voltage();
+    QLabel& setBatteryCMU3Cell6Voltage();
+    QLabel& setBatteryCMU3Cell7Voltage();
+    QLabel& setBatteryCMU3Cell8Voltage();
+
+    QLabel& setBatteryCMU4Temp();
+    QLabel& setBatteryCMU4Cell1Voltage();
+    QLabel& setBatteryCMU4Cell2Voltage();
+    QLabel& setBatteryCMU4Cell3Voltage();
+    QLabel& setBatteryCMU4Cell4Voltage();
+    QLabel& setBatteryCMU4Cell5Voltage();
+    QLabel& setBatteryCMU4Cell6Voltage();
+    QLabel& setBatteryCMU4Cell7Voltage();
+    QLabel& setBatteryCMU4Cell8Voltage();
+
+    QList<QLabel*>& batteryCMUCellVoltageLabels();
+
+    QTextEdit& historyTextEdit();
+
+    QLabel& setInputSerialConnectionStatus();
+    QLabel& setOutput3GConnectionStatus();
+
+private:
+    double mppt1CurrentIn;
```

Comment:
So far within this project all private class members have an underscore following the name. I think it would be a good thing to keep consistent.